next.config.js

```diff
@@ -281,6 +281,11 @@ const nextConfig = {
         permanent: true,
       },
       // Reference redirects
+      {
+        source: "/sdks/:sdk",
+        destination: "/in-app-ui/:sdk/sdk/overview",
```

Comment:
```suggestion
        destination: "/in-app-ui/:sdk/overview",
```
I think it should be like this? i.e. https://docs.knock.app/in-app-ui/react/overview

Reply:
first `:sdk` is the SDK name, like "react" or "android"

second sdk is just "sdk"
```
https://docs.knock.app/in-app-ui/:sdk/sdk/overview
=== matches ===
https://docs.knock.app/in-app-ui/react/sdk/overview
```